1 table.sql

```diff
@@ -0,0 +1,30 @@
+use MusicNoteDB 
+go 
+
+drop table if exists MusicRental
+go 
+create table dbo.MusicRental(
+    InstrumentId int not null identity primary key,  
+    CustomerFirstName varchar(30) not null constraint c_MusicNote_CustomerFirstName_cannot_be_blank check (CustomerFirstName > ''),
+    CustomerLastName varchar(30) not null constraint c_MusicNote_CustomerLastName_cannot_be_blank check (CustomerLastName > ''),
+    Instrument varchar(20) not null constraint c_MusicNote_Instrument_cannot_be_blank check (Instrument > ''),
+    Category varchar(14) not null constraint c_MusicNote_Category_is_Brass_Wind_Percussion_or_String check(Category in ('Brass', 'Wind', 'Percussion', 'String')),
+    MonthlyRentalFee decimal(4,2) not null constraint c_MusicNote_MonthlyRentalFee_greater_than_0 check (MonthlyRentalFee > 0),
+    DateRented date not null constraint c_MusicNote_DateRented_cannot_be_blank check(DateRented > '') ,
```

Comment:
No need for blank constraint on date, DateRented should  be before the current date.

Reply:
Removed this and replaced it with DateRented cannot be future date